### PR TITLE
Add mods for debian12 workaround

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Comment by fernandohds564** [comment]
why is this PR not ready for review yet?

**Comment by VitorSouzaLNLS** [comment]
I made a suggestion-commit directly in the script. I tried to create only a git comment, but i couldnt... sorry.

**Comment by xresende** [comment]
@fernandohds564 , can you review it so that it can be merged ?

**Comment by fernandohds564** [comment]
I hadn't reviewed because it was not marked as ready for review.

File: bin/sirius-script-mamba-env-create.bash [hunk 1]
```diff
@@ -408,7 +406,7 @@ then
     printf_yellow_clear "Installing accelerators simulation packages.\n"
     clone_or_find lnls lnls-fac && make $TARGET
     clone_or_find trackcpp lnls-fac && make clean && \
-        make install-cpp 2>/dev/null && make $TARGET-py 2>/dev/null
+        make install-cpp 2>/dev/null && make install-py 2>/dev/null
```
Comment (fernandohds564) [inline bin/sirius-script-mamba-env-create.bash:409]: I think we could remove this redirection of errors to/de/null. What do you think?

**Review by fernandohds564 — CHANGES_REQUESTED** [review]

File: bin/sirius-script-mamba-env-create.bash (hunk 1 above)
Comment (xresende) [reply bin/sirius-script-mamba-env-create.bash]: done

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**